db/migrate/20160203195736_remove_docker_image_schema.rb

```diff
@@ -0,0 +1,39 @@
+class RemoveDockerImageSchema < ActiveRecord::Migration
```

Comment:
I don't know if we need to remove the V1 docker repos as a rake task, or if we need to do it here.